src/crypting.c

```diff
@@ -54,7 +54,12 @@ static const UInt4 rinit[] = {
     0x6a09e667, 0xbb67ae85, 0x3c6ef372, 0xa54ff53a,
     0x510e527f, 0x9b05688c, 0x1f83d9ab, 0x5be0cd19 };
 
-/* TODO: These depend on endianness */
+#ifdef BIGENDIAN
```

Comment:
Any reason not to use WORDS_BIGENDIAN, defined by GAP?

Reply:
None.  I didn't realize that existed.  Good, that means that neither configure nor Makefile.in needs to be touched at all.  Let me do another test build to be sure I've got it right and I'll push a revised commit.